### ISSUE TITLE
Backend: useful responses to frontend requests

### ISSUE THREAD

**Issue by annikakrause** [amosproj/amos2022ss04-digital-identity#99]
## User story

1. As a {developer}
2. I want / need {useful responses from the backend to frontend requests}
3. So that {I can implement error handling in the frontend}

## Acceptance criteria

- respond to requests in response format with header and body (no plain Strings!)
- add error handling with the common and useful http codes
- add the http codes and the implemented behavior to our open API document

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release

**Comment by annikakrause** [comment]
@TamaraHoock  @veitmo this could be a good additional issue for sprint 6

**Comment by Idontker** [comment]
Done in sprint 5 https://github.com/amosproj/amos2022ss04-digital-identity/commit/8b9cdf974b0592bb76a4d87a3d35fe9834c38272
